src/components/CharactersPage/CharacterForm/CharacterForm.js

```diff
@@ -212,12 +237,11 @@ const CharacterForm = ({ onClose }) => {
             >
               <ControlledNumberInputField
                 control={control}
+                error={errors.height}
```

Comment:
Again be mindful of the expected data types expected by all your functions and components. If they expect a string, give a string, if they expect a boolean, give a boolean